IdentityCore/src/network/response_serializer/MSIDHttpResponseSerializer.m

```diff
@@ -30,7 +30,20 @@ - (id)responseObjectForResponse:(NSHTTPURLResponse *)httpResponse
                         context:(id <MSIDRequestContext>)context
                           error:(NSError **)error
 {
-    return data;
+    id result;
```

Comment:
nit: do we even need this variable?

Reply:
I prefer to keep it. In my opinion, it looks weird if we have returns inside "if" but there is no return at the end of the method.